vNext/Orleans.vNext.sln

```diff
@@ -422,184 +405,172 @@ Global
 		{DF733504-A366-4860-813E-D20AABE243B2}.Release|x86.Build.0 = Release|Any CPU
 		{D9718C48-7020-4A0F-ACA0-17B28DCD6339}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
 		{D9718C48-7020-4A0F-ACA0-17B28DCD6339}.Debug|Any CPU.Build.0 = Debug|Any CPU
-		{D9718C48-7020-4A0F-ACA0-17B28DCD6339}.Debug|x64.ActiveCfg = Debug|x64
-		{D9718C48-7020-4A0F-ACA0-17B28DCD6339}.Debug|x64.Build.0 = Debug|x64
-		{D9718C48-7020-4A0F-ACA0-17B28DCD6339}.Debug|x86.ActiveCfg = Debug|x86
-		{D9718C48-7020-4A0F-ACA0-17B28DCD6339}.Debug|x86.Build.0 = Debug|x86
+		{D9718C48-7020-4A0F-ACA0-17B28DCD6339}.Debug|x64.ActiveCfg = Debug|Any CPU
+		{D9718C48-7020-4A0F-ACA0-17B28DCD6339}.Debug|x64.Build.0 = Debug|Any CPU
+		{D9718C48-7020-4A0F-ACA0-17B28DCD6339}.Debug|x86.ActiveCfg = Debug|Any CPU
+		{D9718C48-7020-4A0F-ACA0-17B28DCD6339}.Debug|x86.Build.0 = Debug|Any CPU
```

Comment:
There seem to be a lot of unnecessary configurations. In a future PR we should probably clean those